libs/watt/src/lib/components/tabs/watt-link-tabs.component.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, ViewEncapsulation, contentChildren } from '@angular/core';
+import { MatTabsModule } from '@angular/material/tabs';
+import { RouterLink, RouterLinkActive, RouterOutlet } from '@angular/router';
+import { WattLinkTabComponent } from './watt-link-tab.component';
+
+@Component({
+  standalone: true,
+  selector: 'watt-link-tabs',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
issue: since the component uses `ViewEncapsulation.None`, the styles below will apply globally. This also means to tabs used in drawers even though this might not be desired. 
We've solved this issue in `WattTabsComponent` component by wripping the styles in `watt-tabs`.